### ISSUE TITLE
SharedMemory connection issue on Windows

### ISSUE THREAD

**Issue by gwm17** [ATTPC/Spyral#153]
Seems like the loopback is busted? somehow on windows....

**Comment by gwm17** [comment]
This is fixed, waiting for v0.10.0. For anyone who needs this urgently, the fix can be hacked in by changing the following code in `src/spyral/pipeline.py`:

Change
```py
manager = SharedMemoryManager(("", 50000))
```
to
```py
manager = SharedMemoryManager()
```
Apparently empty string address on the server listener is unsupported on Windows? By not passing arguments we allow the OS to figure it out

**Comment by gwm17** [comment]
Eventually, there are configurations where we may need to support specifying the listener conditions. I.e. if people want to distribute the mesh across multiple machines over a network.
